railties/lib/rails/generators/rails/app/templates/config/cable.yml

```diff
@@ -7,3 +7,7 @@ test:
 production:
   adapter: redis
   url: redis://localhost:6379/1
+  # To avoid channel collisions when multiple use the same Redis instance,
+  # we prefix channel names with the app/environment name.
+  # (https://redis.io/topics/pubsub#database-amp-scoping)
```

Comment:
I think docs around ac adapter configuration needs some work. Don't think it even mentions the existence of a Postgres adapter for example. :)